manifests/yamls/postgresql.yaml

```diff
@@ -208,7 +208,7 @@ spec:
               mountPath: /bitnami/postgresql
               subPath: 
         - name: metrics
-          image: quay.io/devtron/postgres_exporter:v0.4.7
+          image: "quay.io/devtron/postgres_exporter:v0.4.7"
```

Comment:
Also add quotes to postgres image in statefulset

Reply:
Done